webhooks-extension/test/cleanup.sh

```diff
@@ -9,8 +9,11 @@ source ${test_dir}/util.sh
 if [ $DASHBOARD_INSTALL_NS != "default" ]; then
   kubectl delete ns ${DASHBOARD_INSTALL_NS}
 else
-  kubectl delete deployment tekton-dashboard -n $DASHBOARD_INSTALL_NS
+  kubectl delete deployment tekton-dashboard -n default
+  kubectl delete deployment webhooks-extension -n default
```

Comment:
Do the kservices go too? How about the configmap called githubwebhook or any existing githubsource CRDs, or is this a "run me once" type thing?

Reply:
Discussed with Mark, he found it useful while writing this - while it's not the most thorough of cleanups (e.g. you'd end up with lingering webhooks perhaps if you kept running it), nobody else is actually calling this script and it's more there for convenience. I'd rather we keep it as it is (so it's partially useful at least albeit not 100% rock solid), so resolving this convo